src/state/apis/foxy/foxyApi.ts

```diff
@@ -128,12 +126,20 @@ const makeMergedOpportunities = (
   })
 
 async function getFoxyOpportunities(
-  balances: PortfolioBalancesById,
+  state: any,
```

Comment:
any chance of tightening this, pardon the pun?

Reply:
guessing we can't make this `ReduxState` because of circular deps? maybe just a comment `// ReduxState`

Reply:
Precisely ! Comment added in 41703e46a84e7d7a33241b8d1d0580fb887e7315